[2022-12-16 02:21:05,283] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: process_web_log.scan_for_log manual__2022-12-16T02:21:02.774646+00:00 [queued]>
[2022-12-16 02:21:05,297] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: process_web_log.scan_for_log manual__2022-12-16T02:21:02.774646+00:00 [queued]>
[2022-12-16 02:21:05,298] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-16 02:21:05,298] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-16 02:21:05,298] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-16 02:21:05,315] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): scan_for_log> on 2022-12-16 02:21:02.774646+00:00
[2022-12-16 02:21:05,321] {standard_task_runner.py:52} INFO - Started process 110 to run task
[2022-12-16 02:21:05,326] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'process_web_log', 'scan_for_log', 'manual__2022-12-16T02:21:02.774646+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/test1.py', '--cfg-path', '/tmp/tmpaynn_zgy', '--error-file', '/tmp/tmpde6rzk25']
[2022-12-16 02:21:05,327] {standard_task_runner.py:80} INFO - Job 16: Subtask scan_for_log
[2022-12-16 02:21:05,383] {task_command.py:369} INFO - Running <TaskInstance: process_web_log.scan_for_log manual__2022-12-16T02:21:02.774646+00:00 [running]> on host 411836952c8f
[2022-12-16 02:21:05,452] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=process_web_log
AIRFLOW_CTX_TASK_ID=scan_for_log
AIRFLOW_CTX_EXECUTION_DATE=2022-12-16T02:21:02.774646+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-16T02:21:02.774646+00:00
[2022-12-16 02:21:05,453] {logging_mixin.py:115} INFO - scanning a folder 
[2022-12-16 02:21:05,455] {logging_mixin.py:115} INFO - scanning a folder finished
[2022-12-16 02:21:05,456] {python.py:173} INFO - Done. Returned value was: None
[2022-12-16 02:21:05,470] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=process_web_log, task_id=scan_for_log, execution_date=20221216T022102, start_date=20221216T022105, end_date=20221216T022105
[2022-12-16 02:21:05,498] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-16 02:21:05,553] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
